### PR TITLE
Basic code cleanup

### PR DESCRIPTION
**Pull request by bocekm**
- replaced using six library by python built-in functions
  - the code without six is still python 2.x and 3.x compatible
- remove unused imports, commented out code
- wrap code to 80 chars
- functionality is kept the same

### CODE REVIEW

**Comment by pirat89** [comment]
I gues that some commits can be dropped. E.g. for python-2.4 patches.

**Comment by bocekm** [comment]
Waiting for https://github.com/upgrades-migrations/preupgrade-assistant/pull/258 to be merged to master. Then I'll do a rebase and thus remove deprecated patches.

File: preupg/kickstart/plugins/partitioning.py [hunk 1]
```diff
@@ -182,21 +202,29 @@ def get_volume_info(filename, first_index, second_index):
     def get_partition_layout(self, lsblk, vgs, lvdisplay):
         """
         Returns dictionary with partition and realname and size
-        :param filename:  filename with partition_layout in /root/preupgrade/kickstart directory
+        :param filename:  filename with partition_layout in
+                          /root/preupgrade/kickstart directory
```
Comment (pirat89) [inline preupg/kickstart/plugins/partitioning.py]: is it really up-to-date description? see header of func.

File: preupg/application.py [hunk 2]
```diff
@@ -35,10 +34,12 @@
 
 
 def fault_repr(self):
-    """monkey patching Fault's repr method so newlines are actually interpreted"""
+    """monkey patching Fault's repr method so newlines are actually interpreted
+    """
```
Comment (bocekm) [inline preupg/application.py]: That's against documentation convention (https://www.python.org/dev/peps/pep-0257/)

Comment (bocekm) [inline preupg/application.py]: I'll change it to
```
"""Monkey patching Fault's repr method to interpret newlines."""
```

File: preupg/kickstart/plugins/partitioning.py (hunk 1 above)
Comment (bocekm) [inline preupg/kickstart/plugins/partitioning.py]: Most probably isn't :) I changed it to have each line less then 80 chars, but you're right, I'll update the content to reflect the current params.

**Comment by AloisMahdal** [comment]
Better late than never --- I looked at the diff and all seems OK.

I also ran few downstream tests with builds from your private COPR and all passed.

**Pull request merged**